### ISSUE TITLE
User registration password validation message not showing

### ISSUE THREAD

**Issue by frankhale** [DIOS-Tech/BlueHarvest#58]
We've regressed! User registration is broken.

**Comment by frankhale** [comment]
It's not really broke but there is a regression. A message is supposed to be displayed on the page if you don't form your password correctly. I needed an uppercase character. It used to display the message but now it's not. It displays the messages for other invalid states.

**Comment by frankhale** [comment]
Seq.map is lazy, not sure why I didn't change this earlier. We'll just use Seq.iter to add errors to the ModelState.

```fsharp
member private this.AddErrors(result:IdentityResult) =
	result.Errors
	|> Seq.iter (fun error -> this.ModelState.AddModelError("", error))
	|> ignore
```

**Comment by frankhale** [comment]
Fixed via ec414a65be3771db912bb1cf7f17653f2fa9bc2b
